lib/streamlit/server/server.py

```diff
@@ -124,11 +129,27 @@ def start_listening(app):
 
     """
 
-    call_count = 0
     http_server = tornado.httpserver.HTTPServer(
         app, max_buffer_size=config.get_option("server.maxUploadSize") * 1024 * 1024
     )
 
+    if server_address_is_unix_socket():
+        start_listening_unix_socket(http_server)
+    else:
+        start_listening_tcp_socket(http_server)
+
+
+def start_listening_unix_socket(http_server):
+    address = config.get_option("server.address")
+    file_name = os.path.expanduser(address[7:])
```

Comment:
Can you please add a comment here explaining that we are just dropping the `unix://` prefix?

Reply:
Thanks for the PR, btw!

Reply:
Instead of a comment, I could also imagine it as self-documenting code: `address = address[len("unix://"):]`. Up to you on whatever seems most readable!

Reply:
I think I would like to create a 'constant' `UNIX_SOCKET_PREFIX = "unix://"` which can be used at https://github.com/streamlit/streamlit/pull/1802/files#diff-04c5f86a3af3ad246da39ab2c237bee4R122 and https://github.com/streamlit/streamlit/pull/1802/files#diff-04c5f86a3af3ad246da39ab2c237bee4R144 . What do you think?